src/main/java/org/opensearch/ad/util/IndexUtils.java

```diff
@@ -116,17 +126,27 @@ public String getIndexHealthStatus(String indexOrAliasName) throws IllegalArgume
      * @return The number of documents in an index. 0 is returned if the index does not exist. -1 is returned if the
      * request fails.
      */
-    /*  @anomaly-detection - commented until we have support for SDKRestClient.stats() https://github.com/opensearch-project/opensearch-sdk-java/issues/620
-    @Deprecated
```

Comment:
Do we need to migrate a deprecated method?

Reply:
yes it required to be uncommented because the method is been used in ADStatsTests